Java-Data-Types.md

```diff
@@ -0,0 +1,148 @@
+# Java data-Types
+
+Java is a strongly ­typed language. This means that, in Java, each data type has its own strict definition. There are no implicit data type conversions when any conflicts occur between the data types. Any change in data types should be explicitly declared by the programmer.
+ 
+Java defines 8 simple data types. They are divided into the following categories:
+ 
+Integers
+ 
+Floating­ Point Numbers
+ 
+Characters
+ 
+Boolean Type
+ 
+The details of each of the data types is given below
+ 
+### Integers:
+
+These are of four types: byte, short, int, long. It is important to note that these are signed positive and negative values. Java does not support positive­ only numbers.
```

Comment:
You are writing in markdown, so use the power of that judiciously.

> These are of four types: byte, short, int, long

should be

> These are of four types: `byte`, `short`, `int`, `long`

Also note that you are listing only the  primitive types. I think you should mention the proper wrapper classes along with them as well.